ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-rhoai.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.21
+    tag: golang-1.22
```

Comment:
this will be needed, but need wait till ODH PR land to rhoai branch

Reply:
@zdtsw I'm confused by the chicken-egg-ness of this: I thought that this would be needed on any branch _before_ those branches would be able to start using the newer go, or else the tests wouldn't be able to pass?

Reply:
what i meant is:
we could have new commits to rhoai branch before we get the 1.22 PR to rhoai.
(if we are going step by step: merge 1.22 to incubation, test it,  days after.............., sync the PR to rhoai)
during these "days between", e2e from this PR uses 1.22 to build our code, in theory, they should work

Reply:
Ah right, understood, thanks :+1: I've removed it from here for now then